frontend/src/components/Tag.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { type Friend } from './PlanForm'
+import './tag.css'
+
+interface Props {
+  friend: Friend
+  handleRemoveTag: (friend: Friend) => void
+}
+
+export const Tag: React.FC<Props> = ({ friend, handleRemoveTag }) => {
+  const [backgroundColor] = useState<any>(generateRandomLightColor())
```

Comment:
does this need to be a state variable? you can simply just declare this variable and use it directly without setting state

Reply:
Oh this scenario was pretty interesting. Let say I use `const backgroundColor = generateRandomLightColor()` then every time I render the `Tag` component, it will also regenerate a new color in the UI. But I use `useState`, the color will not regenerate when the `Tag` component re-renders.